cli.js

```diff
@@ -142,6 +142,8 @@ process.on('uncaughtException', function(err) {
 
     flags = flags || [];
     optFlags = optFlags || [];
+    // we always take a log option
+    optFlags.push('log');
```

Comment:
Ah right of course. I know it's not ideal, but perhaps we just need to strip out the `--log` argument manually instead of running `readOptions` twice, something like:

```javascript
  var logArg = args.indexOf('--log');
  if (logArg > -1) {
    ui.setLogLevel(args[logArg + 1]);
    args.splice(logArg, 2);
  }
```